scripts/format.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+import multiprocessing
+import os
+import sys
+from functools import partial
+from subprocess import STDOUT, CalledProcessError, check_output
+
+from termcolor import cprint
+
+import b
+
+
+def _do_format(path):
+    text = ""
+    try:
+        # Use the absolute path to file
+        abs_path = os.path.abspath(path)
+
+        if path.endswith((".h", ".c", ".cc", ".cxx", ".cpp", ".hpp", ".ipp")):
```

Comment:
We should look for `proto` files that are not in the `protos` folder and format them as well